selftests/unit/test_xunit.py

```diff
@@ -39,7 +39,7 @@ def test(self):
 
         self.tmpfile = tempfile.mkstemp()
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
-        args = argparse.Namespace(logdir=self.tmpdir)
+        args = argparse.Namespace(base_logdir=self.tmpdir)
```

Comment:
Ok. I agree we don't really need a workaround for plugins, since we maintain an LTS.